packages/features/form-builder/utils.ts

```diff
@@ -67,3 +70,18 @@ export const getVariantsConfig = (field: Pick<z.infer<typeof fieldSchema>, "vari
   }
   return variantsConfig;
 };
+
+export function getVariantFieldLabel({
+  variantField,
+  fieldTypeVariantConfig,
+  t,
+}: {
+  variantField: VariantConfig["fields"][number];
+  fieldTypeVariantConfig: FieldTypeVariantConfig | undefined;
+  t: TFunction;
+}) {
+  if (!variantField.label?.trim()) {
```

Comment:
Your logic is here now. Also removed length check, it's a string and we can simply check for truthy.